lib/Model.js

```diff
@@ -67,9 +67,14 @@ class Model {
 
     const advisoriesFound = this.advisoriesFound.map(advisoryMapper);
 
+    const whitelistedAdvisoriesNotFound = this.whitelistedAdvisoryIds.filter(
+      id => this.whitelistedAdvisoriesFound.indexOf(id) === -1
```

Comment:
If we're not supporting node < 6 we can `!this.whitelistedAdvisoriesFound.includes(id)`

Reply:
Good call. No need to support < 6 since `npm audit` was introduced in 6.x.